imports/plugins/included/taxes-taxcloud/client/settings/taxcloud.js

```diff
@@ -20,8 +21,21 @@ Template.taxCloudSettings.helpers({
 AutoForm.hooks({
   "taxcloud-update-form": {
     onSuccess: function () {
-      return Alerts.toast(i18next.t("admin.taxSettings.shopTaxMethodsSaved"),
-        "success");
+      Meteor.call("taxcloud/getTaxCodes", (err, res) => {
+        if (err) {
+          throw new Meteor.Error(Number, "description");
```

Comment:
Why are you are throwing a JS type as the first parameter to `Meteor.Error`?

Reply:
Will fix that , I was not sure the error code I wanted to use at that time so this is an oversight.